src/main/java/hudson/plugins/junitattachments/AttachmentPublisher.java

```diff
@@ -33,7 +33,7 @@
     public AttachmentPublisher() {
     }
 
-    public static FilePath getAttachmentPath(AbstractBuild<?, ?> build) {
+    public static FilePath getAttachmentPath(Run<?, ?> build) {
```

Comment:
Not binary compatible, don't you need to retain the signature with AbstractBuild (deprecated)?